resets/resets.js

```diff
@@ -1,4 +1,5 @@
 export const resets = `
+/* layer: preflights */
```

Comment:
Just curious, since we're adding this, is it still necessary to apply these resets in [@warp-ds/drive](https://github.com/warp-ds/drive/pull/106/files#diff-19d070610e43a72b5fc9100d8258043bd9e2447cdc27732544de0994a445def6R1-R4) with a `layer: 'preflights'` setting?
```
export const resets = {
  layer: 'preflights',
  getCSS: async () => (await fetch('https://assets.finn.no/pkg/@warp-ds/css/v1/resets.css')).text(),
};

Reply:
Ah that we shouldn't add here. Good catch, I'll remove

Reply:
Fixed here https://github.com/warp-ds/css/pull/2/commits/4f736574218005c2e64c524f333954c1f272dfb7